py/torch_tensorrt/dynamo/conversion/impl/normalization/ops.py

```diff
@@ -46,8 +46,20 @@ def batch_norm(
     if has_dynamic_shape(input.shape):
         assert input.shape[1] != -1, "Channel dim can't be dynamic for batch norm."
 
+    if weight is None:
+        weight = 1.0
+
+    if bias is None:
+        bias = 0.0
+
+    if running_mean is None:
+        running_mean = 0.0
+
+    if running_var is None:
+        running_var = 1.0
+
     scale = cast(torch.Tensor, to_numpy(weight)) / np.sqrt(
-        cast(torch.Tensor, to_numpy(running_var)) + cast(float, eps)
+        cast(torch.Tensor, to_numpy(running_var)) + eps
```

Comment:
The `torch.Tensor` cast can be removed, because `to_numpy` will return an `np.ndarray`, so this typing would be incorrect.